net/kea/Makefile

```diff
@@ -70,7 +70,7 @@ endef
 define Package/kea-ctrl
 	$(call Package/kea/Default)
 	TITLE+=Control
-	DEPENDS:=+kea-dhcp4 \
+	DEPENDS:=+procps-ng-ps +procps-ng +kea-dhcp4 \
```

Comment:
procps-ng dependency order should be swapped for consistency.

Reply:
Ok, should I make another commit or roll back and fix that commit?

Reply:
Also fixed in the latest commits